### DISCUSSION TITLE
Fully customize optuna visualization

### DISCUSSION THREAD

**Discussion by askppp** [optuna/optuna#4613] [Q&A] [answered]
Hi and thank you for this repo,

i'm making some optimizations and i want to be able to generate high quality images for them at the end.

I saw that the "optuna.visualization" module can help with that. 

But i need to do this following changes and not have to change the source code under "optuna/optuna/visualization/":

* Change the font type
* Change the color of the dots and the lines
* change the color of the bars

Thanks.

**Comment by nzw0301** [comment, marked as answer]
I think you can do that since optuna visualisation returns an editable object.
